components/econet/econet.h

```diff
@@ -74,7 +77,23 @@ class Econet : public Component, public uart::UARTDevice {
 
   void set_src_address(uint32_t address) { src_adr_ = address; }
   void set_dst_address(uint32_t address) { dst_adr_ = address; }
-  void set_update_interval(uint32_t interval_millis) { update_interval_millis_ = interval_millis; }
+  void set_mod_req_updates_(std::vector<uint8_t> keys, std::vector<uint32_t> values) {
+    for (auto i = 0; i < keys.size(); i++) {
+      request_mod_interval_millis_[keys[i]] = values[i];
+      request_mod_last_requested_[i] = i * values[i] / MAX_REQUEST_MODS;
```

Comment:
It is.  

The thought was to space slow requests out a big further than the min interval time by giving it a bit more of an offset. Ie for 1x 2 second request group and 3x 30 second request groups. With them starting at zero, they will all be wanting to request right at 30 seconds after start with a burst as opposed to spacing them out ~every 10 seconds. 

This can be a future enhancement.